yaksh/models.py

```diff
@@ -421,6 +435,8 @@ def get_completion_status(self, user, course):
         if course_status.exists():
             if self in course_status.first().completed_units.all():
                 state = "completed"
+            elif self.type == "quiz":
```

Comment:
So the issue is when quiz is finished and the user does not click the next learning unit, the learning unit is not considered as completed, but the Quiz is complete.
Am I correct about the issue?

If yes then, try if possible to update learning unit along with the completion of Quiz!

PS: Please refer issue number in the PR

Reply:
Can you suggest where can I add a quiz to completion unit?

Reply:
Resolved. Now quiz completion will depend on the answer paper status `in progress` or `completed`.